src/Framework/AbstractConfig.php

```diff
@@ -20,7 +20,14 @@ protected function get(string $key, $default = Config::DEFAULT_CONFIG_VALUE)
         return Config::getInstance()->get($key, $default);
     }
 
-    protected function getAppRootDir(): string
+	/**
+	 *
+	 * Allow easy access the root dir of the project from the Factory
```

Comment:
Maybe something like this would be better, what do you think?
```suggestion
	 * Allow easy access to the root dir of the project.
```